src/3p-analytics-common.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @enum {string} */
+export const AMP_ANALYTICS_3P_MESSAGE_TYPE = {
+  EVENT: 'E',
```

Comment:
@keithwrightbos Your thoughts? IIRC you said to make it a single char, to minimize # of bytes sent over the wire. Given that the other 3 enum values are gone, can we just make this a const instead of an enum?